iree/compiler/Conversion/LinalgToLLVM/HALInterfaceToMemrefArguments.cpp

```diff
@@ -122,6 +127,11 @@ struct ProcessFuncInterfacePattern : public OpConversionPattern<FuncOp> {
         ++argIndex;
       }
     }
+    SmallVector<int64_t, 4> shapes;
```

Comment:
Could you put a comment here to mention that "Use the last function argument to pass in all dynamic dimensions"?